README.md

```diff
@@ -431,12 +432,12 @@ Table of Contents
   * [Continuous PHP](https://continuousphp.com/) — continuousphp is the first and only PHP-centric Platform to build, package, test and deploy applications in the same workflow. Free for Community Projects i.e. OSS/Public/Educational projects.
   * [deployhq.com](https://www.deployhq.com/) — 1 project with 10 daily deployments (30 build minutes/month)
   * [drone](https://cloud.drone.io/) - Drone Cloud enables developers to run Continuous Delivery pipelines across multiple architectures - including x86 and Arm (both 32 bit and 64 bit) - all in one place
+  * [Jfrog Pipelines (formerly Shippable)](https://jfrog.com/start-free) — The free tier offers 2,000 CI/CD minutes per month at no cost.
```

Comment:
Combine to one line please, we can list fewer languages in the main repo and mention it includes CI/CD